ui/lib/core/addon/components/text-file.hbs

```diff
@@ -6,7 +6,7 @@
 {{#unless @uploadOnly}}
   <div class="level is-mobile">
     <div class="level-left">
-      <label for="input-{{this.elementId}}" class="has-text-weight-semibold" data-test-text-file-label>
+      <label id="text-file-input-label" class="has-text-weight-semibold" data-test-text-file-label>
```

Comment:
https://rfcs.emberjs.com/id/0659-unique-id-helper/

this is a nice syntax/helper to add IDs directly in the template

Reply:
Good point, I do want the ID's to be unique. Now that the other tests are somewhat stable I'll put the elementID back and check whether it's still failing

Reply:
Update: it was originally failing in one test, when `uploadOnly` is passed which does not render the label. In other places I tried to make the input ID's deterministic so that if we do have a label outside of the input, we can add `for="the-id"` to the label and it connects it, but when we use random ID's that's not possible. I think this is something to think about for our form refresh, but for now I just added a screenreader-only label when `uploadOnly` is true.